plugins/ecue.js

```diff
@@ -219,7 +219,7 @@ function exportHandleChannel(fixture, mode, dmxCount, viewPosCount, fineChannels
   dmxByte1++;
 
   if (!('name' in channel)) {
-    channel.name = chKey;
+    channel.name = mode.channels[dmxCount];
```

Comment:
This is not how the model would handle it. Shouldn't we try to stick to that behavior?